src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -67,7 +64,7 @@ const Conversation: React.FC<ConversationProps> = props => {
     }
   }
 
-  useEffect(scrollToBottom, [conversation, lastMessageID])
+  useEffect(scrollToBottom, [conversation])
```

Comment:
@ivan-badyulya this actually triggers that on loading more messages on infinite scroll, every time, the user will be scrolled to the bottom instead of where they were. Please revert anything related to `lastMessageID`.

Reply:
@jo-rs sorry, I tried to understand why does it track the last ID but didn't think about infinite scroll. I wanted to ask you, but you were on vacation. Moved it back

Reply:
@jo-rs yes, I did a demo last week to discuss this stuff. The thing is the scrollIntoView works before some child component rerender and impact the scrolling. 1 stuff I noticed the wierd scrolling behavior is when we added the new make offer button. The button renders after scrollIntoView it impacts the message container height that's why scrolling wasn't working all the way down, that's why I execute the scrollToBottom on makeOfferButton did mount.

Bot now as I see the thing is not only in make offer button

Reply:
@jo-rs i tried to fix the stuff in the [pr ](https://github.com/artsy/force/pull/7932)